customScripts/expansiontravel.gd

```diff
@@ -2,17 +2,21 @@
 extends Node
 
 func randomfromarray(array):
-	return array[randi() % array.size()]
+	if array.size() > 0:
+		return array[randi() % array.size()]
+	else:
+		return []
```

Comment:
Why return an array? This makes no sense to me. Returning null at least fits standard expectations.

Reply:
My bug fixing involves launching spaghetti at a wall apparently.  It did work, but may have made a mess in the process.  -_-

Reply:
comment incorporated on new commit